cw-localsys

```diff
@@ -316,7 +316,6 @@ on_gentoo () {
 		;;
 	pkg-latest)
 		NAME=$1
-		/usr/bin/eix-sync -q
```

Comment:
Should this be moved to pkg-recache, and replace the "FIXME lolwut?" comment there?